package.json

```diff
@@ -60,18 +61,18 @@
     "parser": "babel-eslint"
   },
   "scripts": {
-    "lint": "node node_modules/standard/bin/cmd.js src/**/*.js",
```

Comment:
Sorry that was force of habit, npm automatically adds node_modules to your path so you don't actually need those libraries installed globally. It also has greater compatibility for Windows developers 

> On 23 Oct 2015, at 16:53, Jason Dobry <notifications@github.com> wrote:
> 
> In package.json:
> 
> > @@ -60,18 +61,18 @@
> >      "parser": "babel-eslint"
> >    },
> >    "scripts": {
> > -    "lint": "node node_modules/standard/bin/cmd.js src/**/*.js",
> Why did you change these commands? Now users will have to install these 5 libraries globally in order to run the build. I'd prefer devs be able to just clone, npm install and npm run all without any additional setup.
> 
> —
> Reply to this email directly or view it on GitHub.
>